man/systemd.network.xml

```diff
@@ -3153,6 +3153,28 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
         3600 seconds (one hour).</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>HomeAgent=</varname></term>
+
+        <listitem><para>Takes a boolean. Specifies that IPv6 router advertisements which indicates to hosts that
+        the router acts as a Home Agent and includes a Home Agent Option. Defaults to false.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>MobileAgentLifetimeSec=</varname></term>
+
+        <listitem><para>Takes a timespan. Specifies the lifetime of the Home Agent. An integer the default unit of seconds,
+        in the range 0...65520. Defaults to 0 seconds means, to place the current Router Lifetime value.</para>
```

Comment:
`0...65520.` -> `0…65520 seconds.`

BTW, 65520 seconds sound spurious. Could you give me a reference ?

Reply:
I should have added that . https://datatracker.ietf.org/doc/html/rfc6275.html#page-64
```
Home Agent Lifetime

      16-bit unsigned integer.  The lifetime associated with the home
      agent in units of seconds.  The default value is the same as the
      Router Lifetime, as specified in the main body of the Router
      Advertisement.  The maximum value corresponds to 18.2 hours
```

18.2 hrs = 65520 seconds

Reply:
I guess 65535 seconds is rounded to 18.2 hours. It does not mean we will ignore 65521…65535 seconds.

Reply:
Hm... It seems for me that these implementations misunderstand the RFC. Or in other words, the statement of the RFC is quite ambiguous, as both 65535 seconds and 65520 seconds are 18.2 hours.
Are there any erratum of the RFC ??

Reply:
none I know of